servers/services/src/main/java/org/projectnessie/services/config/ServerConfig.java

```diff
@@ -31,4 +33,8 @@ public interface ServerConfig {
    * @return {@code true} if the server should send the stack trace to the client.
    */
   boolean sendStacktraceToClient();
+
+  default int accessChecksBatchSize() {
```

Comment:
just wondering: why are the other defaults not part of the interface?

Reply:
Good question - IDK - those were actually there before me